internal/pkg/pkg.go

```diff
@@ -36,8 +36,12 @@ const ParentDir = ".."
 
 // Pkg represents a kpt package with a one-to-one mapping to a directory on the local filesystem.
 type Pkg struct {
-	UniquePath  types.UniquePath
-	DisplayPath types.DisplayPath
+	UniquePath        types.UniquePath
+	DisplayPath       types.DisplayPath
+
+	// RootPkgUniquePath is the unique path of the root package on which the command is invoked
+	// this must be passed on to the subpackages
+	RootPkgUniquePath types.UniquePath
```

Comment:
you probably want to change `RelativePathTo` since the user doesn't need to pass the parent package to that method.

Reply:
It is unclear to me what we need this. It seems like we should be able to figure out the `UniquePath` and `DisplayPath` of a subpackage simply using the information in the parent package.

Reply:
Changed it to `rootPkgParentDirPath` and made it private. This can just be a string as it is not exposed. Updated the description.

@mortent We can't. We need the information about the root package on which command is invoked. If there are 3 packages `foo/bar/baz`, `bar` should have the information regarding `foo` to determine the display path of `baz`.